### PR TITLE
[AN-5853] Add new event for team creation from profile page

### PR DESCRIPTION
**Pull request by CalumMcCall**
#### APK
[Download build #10006](http://192.168.10.18:8080/job/Pull%20Request%20Builder/10006/artifact/build/artifact/wire-dev-PR1345-10006.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/tracking/GlobalTrackingController.scala [hunk 1]
```diff
@@ -103,24 +103,36 @@ class GlobalTrackingController(implicit inj: Injector, cxt: WireContext, eventCo
...
+               |trackEvent: ${eventArg.name}
+               |properties: ${eventArg.props.map(_.toString(2))}
                |superProps: ${mixpanelGuard.withApi(_.getSuperProperties).getOrElse(sProps).toString(2)}
           """.stripMargin)
         }
       }
 
       event match {
+        case _: OpenedTeamRegistration =>
+          trackingEnabled.map {
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/tracking/GlobalTrackingController.scala]: a flatmap might be slightly clearer here, but I guess we don't care about the result

File: app/src/main/scala/com/waz/zclient/tracking/GlobalTrackingController.scala [hunk 2]
```diff
@@ -89,10 +89,10 @@ class GlobalTrackingController(implicit inj: Injector, cxt: WireContext, eventCo
     */
   ZMessaging.globalModule.map(_.trackingService.events).foreach {
     _ { case (zms, event) =>
-      def send() = {
+      def send(zmsArg: Option[ZMessaging], eventArg: TrackingEvent) = {
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/tracking/GlobalTrackingController.scala]: seems unnecessary having extra arguments here? `zms` and `event` will be immutable

File: app/src/main/scala/com/waz/zclient/tracking/GlobalTrackingController.scala [hunk 3]
```diff
@@ -103,24 +103,36 @@ class GlobalTrackingController(implicit inj: Injector, cxt: WireContext, eventCo
...
           """.stripMargin)
         }
       }
 
       event match {
+        case _: OpenedTeamRegistration =>
+          trackingEnabled.map {
+            case true => ZMessaging.currentAccounts.loggedInAccounts.head.map {
+              _.size match {
+                case 1 =>
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/tracking/GlobalTrackingController.scala]: We may allow any number of simultaneously logged-in accounts, not just 2, so `case 0` should send the original event, and `case _` should send the `FromProfile` event.

**Comment by konradwire** [comment]
no bug, everything works as expected

**Comment by someoneEsle** [comment]
I'm removing the QA label even though this was tested by Konrad -- he doesn't have permissions to remove the label.

**Pull request merged**